spec/factories/users.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    # don't freak out if the database isn't created yet
+    # http://stackoverflow.com/a/27745684/358804
```

Comment:
The problem is that it's trying to `SELECT COUNT(*) FROM users;` when loading the `test` environment, even if the database/table doesn't exist yet.

```
$ RAILS_ENV=test rake db:drop
$ RAILS_ENV=test rake db:setup
rake aborted!
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "users" does not exist
LINE 5:                WHERE a.attrelid = '"users"'::regclass
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"users"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
...
/Users/aidanfeldman/dev/18F/c2/spec/factories/users.rb:3:in `block (2 levels) in <top (required)>'
...
```

Weirdly, `RAILS_ENV=test rake db:schema:load` works, which is what CircleCI uses, and thus why the tests are passing. Not sure of the best way to handle this.